Common subdirectories: libgeotiff-1.4.2_orig/bin and libgeotiff-1.4.2/bin
Common subdirectories: libgeotiff-1.4.2_orig/cmake and libgeotiff-1.4.2/cmake
diff -urN libgeotiff-1.4.2/csv/csv2c.py libgeotiff-1.4.2_orig_new/csv/csv2c.py
--- libgeotiff-1.4.2_orig/csv/csv2c.py	2020-12-28 11:52:16.701546286 +0100
+++ libgeotiff-1.4.2/csv/csv2c.py	2016-08-11 17:40:14.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 #******************************************************************************
 #  $Id: csv2c.py 2472 2014-07-30 19:13:04Z hobu $
 # 
diff -urN libgeotiff-1.4.2/csv/csv_tools.py libgeotiff-1.4.2_orig_new/csv/csv_tools.py
--- libgeotiff-1.4.2_orig/csv/csv_tools.py	2020-12-28 11:52:09.129658480 +0100
+++ libgeotiff-1.4.2/csv/csv_tools.py	2016-08-11 17:40:12.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
 #******************************************************************************
 #  $Id: csv_tools.py 2641 2015-03-01 14:06:18Z rouault $
 # 
